command/app.go

```diff
@@ -77,9 +77,9 @@ var app = &cli.App{
 			NoVerifySSL: noVerifySSL,
 		}
 
-		storage.SetS3Options(s3opts)
+		err := storage.Init(s3opts)
```

Comment:
``` return storage.Init(s3opts) ```

Reply:
I prefer leaving it as `return err`

`return storage.Init(s3opts)` makes me think that the function is returning an initialized storage :)

Reply:
hmm. you are right :)

Reply:
But `return storage.Init(s3opts)` guarantees that the last step in function is initializing the storage and avoids scenarios like:
```go
err := storage.Init(s3opts)

// this function should not be called if err!=nil
another_function()

return err
```